src/components/ToyList.js

```diff
@@ -1,34 +1,43 @@
-import React, { useState } from "react";
-import { Link } from "react-router-dom";
-import { Container, Card, Row, Col, Button } from "react-bootstrap";
-import { sortToys, filterReview } from "../helpers/toyHelpers";
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Card, Row, Col, Button } from 'react-bootstrap';
+import { sortToys, filterReview, searchToys } from '../helpers/toyHelpers';
+import Search from './SearchToys';
 
 const ToyList = ({ toys }) => {
-  const [sortDirection, setSortDirection] = useState("forward");
-  const [reviewFilter, setFilter] = useState("off");
+  const [sortDirection, setSortDirection] = useState('forward');
+  const [reviewFilter, setFilter] = useState('off');
+  const [searchInput, setSearchInput] = useState('');
 
   function HandleSortOptionChange() {
-    setSortDirection("forward");
+    setSortDirection('forward');
   }
 
   function HandleSortOptionChangeBackward() {
-    setSortDirection("backward");
+    setSortDirection('backward');
   }
 
   function HandleFilterWithReviews() {
-    setFilter("on");
+    setFilter('on');
+  }
+
+  function getSearchInput(input) {
+    setSearchInput(input);
   }
 
   function ResetFilter() {
-    setFilter("off");
+    setFilter('off');
   }
 
   let toyArray = toys ? sortToys(toys, sortDirection) : null;
   let filteredToys = filterReview(toyArray, reviewFilter);
+  let searchedToys = searchToys(toys, searchInput);
+  filteredToys = searchedToys;
 
   return (
     <div>
       <div className="alphabeticalDiv">
+        <Search getSearchInput={getSearchInput} />
```

Comment:
Do we need to send the search term to the Search component?  What would things look like if you used redux state, when someone starts typing in the search box it dispatched actions (to update the state)?